chasm/lib/scheduler/scheduler.go

```diff
@@ -523,16 +535,72 @@ func (s *Scheduler) Describe(
 	ctx chasm.Context,
 	req *schedulerpb.DescribeScheduleRequest,
 ) (*schedulerpb.DescribeScheduleResponse, error) {
+	if s.Closed {
+		return nil, ErrClosed
+	}
+
+	visibility := s.Visibility.Get(ctx)
+	memo := visibility.GetMemo(ctx)
+	delete(memo, visibilityMemoFieldInfo) // We don't need to return a redundant info block.
+
+	attributes := visibility.GetSearchAttributes(ctx)
+	delete(attributes, sadefs.TemporalNamespaceDivision)
+	delete(attributes, sadefs.TemporalSchedulePaused)
```

Comment:
These shouldn't come back from visibility... they are system fields, not user fields.
Are you seeing those come back?